CHANGELOG.md

```diff
@@ -4,6 +4,56 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-rc.4](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-rc.4) - 2023-01-30
+
+In this release we fixed bugs and inconsistencies between zcashd and zebrad in the output of the `getblocktemplate` RPC mehod. In addition, we added the proposal mode feature while we continue the effort of adding and testing mining pool RPC methods. 
```

Comment:
```suggestion
In this release we fixed bugs and inconsistencies between zcashd and zebrad in the output of the `getblocktemplate` RPC method. In addition, we added proposal mode to the `getblocktemplate` RPC, while we continue the effort of adding and testing mining pool RPC methods. 
```

Reply:
Thanks, changed in https://github.com/ZcashFoundation/zebra/pull/6045/commits/6862ce4e3dd43e5dab6f87e2d49b0069897f2d60